src/NuGet.Core/NuGet.Packaging/Signing/TrustStore/X509TrustStore.cs

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using NuGet.Common;
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// Represents the X.509 trust store that package signing and signed package verification will use.
+    /// </summary>
+    public static class X509TrustStore
+    {
+        private static IX509ChainFactory Instance;
+        private static readonly object LockObject = new();
+
+        internal static bool IsEnabled { get; }
+
+        static X509TrustStore()
+        {
+            Version DotNet7 = new(major: 7, minor: 0);
+
+            IsEnabled = Environment.Version >= DotNet7;
```

Comment:
This checks the runtime version is that correct? 

But what we really want is to check the SDK version?

It's my understanding that the SDK 7.0 will be targetting the 6.0 runtime, so this check will never be true.

Reply:
@zivkan mentioned that the SDK 7.0 will be on the net7.0 runtime on Linux, so maybe that's what I'm missing. 

Is there a resource that can confirm this.

Reply:
Yet another comment :D https://github.com/dotnet/sdk/blob/69ba492160ae2cf1e5bc6edfc1663a3786240b29/Directory.Build.props#L13 suggests net7.0, but we have an e-mail thread to confirm the behavior.

Reply:
Curious if we can do the check based on the SDK version instead.